### PR TITLE
FIX null eagre entity

### PR DESCRIPTION
**Pull request by Hlavos**

### CODE REVIEW

File: src/MySQL.php [hunk 1]
```diff
@@ -576,7 +576,7 @@ protected function convertDataForExport(array $data, \obo\Carriers\EntityInforma
             }
 
             foreach ($nullEntities as $nullEntity) {
-                $convertedRow = \array_intersect_key($convertedRow, \array_flip(\array_filter(\array_keys($convertedRow), function($key) use ($nullEntity) {return \strpos($key, "_" . $nullEntity. "_") === false;
+                $convertedRow = \array_intersect_key($convertedRow, \array_flip(\array_filter(\array_keys($convertedRow), function($key) use ($nullEntity) {return \strpos($key, $nullEntity. "_") === false;
```
Comment (hAdamm) [inline src/MySQL.php]: Ad space before dot

File: tests/MySQLTest.phpt [hunk 2]
```diff
@@ -266,12 +273,13 @@ class MySQLTest extends \Tester\TestCase {
     public function testDataForQuery() {
         $queryCarrier = $this->createContactQueryCarrier();
         $actualData = $this->storage->dataForQuery($queryCarrier);
+        \var_dump($actualData);
```
Comment (hAdamm) [inline tests/MySQLTest.phpt]: Remove pls

File: src/MySQL.php (hunk 1 above)
Comment (Hlavos) [inline src/MySQL.php]: Fixed

File: tests/MySQLTest.phpt (hunk 2 above)
Comment (Hlavos) [inline tests/MySQLTest.phpt]: Fixed

**Pull request merged**